config/recipes/istio-gateway/README.asciidoc

```diff
@@ -0,0 +1,134 @@
+= Expose Elasticsearch and Kibana using Istio ingress gateway
```

Comment:
Any reason for picking asciidoc instead of Markdown?
Is asciidoc correctly rendered by Github?

Reply:
GitHub has support for AsciiDoc but not the entire spec. I chose AsciiDoc because it's the format we use in our documentation and also because it supports includes and syntax highlighting. With includes, it's easy to write a short explanation about each manifest without having to duplicate the content in two places.